Corgibytes.Freshli.Cli.Test/Extensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Parsing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Corgibytes.Freshli.Cli.Test
+{
+    public static class Extensions
```

Comment:
The convention is to typically name these extension classes like this:

```suggestion
    public static class ParseResultExtensions
```